cmd/zed/agents/zfs_mod.c

```diff
@@ -1279,6 +1290,41 @@ zfs_enum_pools(void *arg)
 	return (NULL);
 }
 
+#define	CONFIG_LINE_MAX 100
+
+static int
+zed_lookup_tunable(const char *zedlet_dir, const char *tuneable,
+    unsigned int *wait_time)
```

Comment:
If this is meant to be generic, maybe rename `wait_time` to `tunable_value` or something
Maybe also name the function for the type, `zed_lookup_uint_tunable()`, incase there are other types in the future